### PR TITLE
Pagination

### LINKED ISSUE #20: Support Client-side Pagination
By default, show rows sorted by the last updated timestamp, and only display the first N rows, create a button to show more data.

### PR DESCRIPTION
**Pull request by sasymer**
Screenshot 2020-07-29 at 11 01 00 AM
Screenshot 2020-07-29 at 11 00 46 AM

- Finish pagination
- Previous/next buttons
- Page number buttons
- Selection to change # rows displayed per page
- Minor refactoring (change variable names in filters.js for more clarity)

### CODE REVIEW

**Review by gagomez — COMMENTED** [review]
Looks like this includes everything in the large PR, can you show what has changed?

**Comment by sasymer** [comment]
> Looks like this includes everything in the large PR, can you show what has changed?

What is the best way to do that?

File: src/main/webapp/table.js [hunk 1]
```diff
@@ -168,4 +197,137 @@ class Table {
...
+
+  addPaginationButtons(thisTableDiv, pageInfoString) {
+    const buttonDiv = document.createElement("div");
+    buttonDiv.id = "button-div-" + this.name;
+
+    const nextButton = document.createElement("button");
+    nextButton.innerHTML = "Next";
+    nextButton.id = "next-button-" + this.name;
+    const id = this.id;
+    nextButton.onclick = function() { nextPage(id); }
```
Comment (gagomez) [inline src/main/webapp/table.js]: I believe you can just call this.nextPage() here. Also try using an arrow funciton `nextButton.onclick = () => this.nextPage()`

**Review by gagomez — APPROVED** [review]

**Review by jiaxinz-g — APPROVED** [review]

File: src/main/webapp/table.js (hunk 1 above)
Comment (sasymer) [reply src/main/webapp/table.js]: Done!

**Pull request merged**